test/e2e/apimachinery/execwithoptions_stress.go

```diff
@@ -0,0 +1,147 @@
+/*
+Copyright 2023 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package apimachinery
+
+import (
+	"bytes"
+	"context"
+	"crypto/rand"
+	"fmt"
+	"sync"
+	"time"
+
+	"github.com/google/go-cmp/cmp"
+	"github.com/onsi/ginkgo/v2"
+
+	v1 "k8s.io/api/core/v1"
+	"k8s.io/kubernetes/test/e2e/framework"
+	e2epod "k8s.io/kubernetes/test/e2e/framework/pod"
+	admissionapi "k8s.io/pod-security-admission/api"
+)
+
+var _ = SIGDescribe("Pod exec", func() {
+
+	f := framework.NewDefaultFramework("execwithoptions-stress")
+	f.NamespacePodSecurityEnforceLevel = admissionapi.LevelBaseline
+	var pod *v1.Pod
+
+	ginkgo.BeforeEach(func(ctx context.Context) {
+		ginkgo.By("creating a pod")
+
+		// Create pod to attach Volume to Node
+		var err error
+		pod, err = e2epod.CreatePod(ctx, f.ClientSet, f.Namespace.Name, nil, nil, false, "")
+		if err != nil {
+			framework.Failf("unable to create pod: %v", err)
+		}
+
+		ginkgo.By("waiting for busybox's availability")
+		err = e2epod.WaitForPodNameRunningInNamespace(ctx, f.ClientSet, pod.Name, f.Namespace.Name)
+		framework.ExpectNoError(err, "wait for busybox pod to be running and ready")
+	})
+
+	ginkgo.It("works under load", func(ctx context.Context) {
```

Comment:
it has to use the tags `[Serial]` and/or `[Disruptive]` https://github.com/kubernetes/kubernetes/pull/115323/files#r1200644164